cpp/src/sort/rank.cu

```diff
@@ -100,7 +100,8 @@ template <typename TieType,
           typename TieBreaker,
           typename Transformer,
           typename TieIterator>
-void tie_break_ranks_transform(rmm::device_vector<size_type> const &dense_rank_sorted,
+void tie_break_ranks_transform(cudf::device_span<size_type const> dense_rank_sorted,
+                               cudf::device_span<TieType> tie_sorted,
```

Comment:
Doc params need to be updated.

Reply:
Not anymore, @karthikeyann 's suggestion allowed me to put it back the way it was.